### PR TITLE
French translation missing

### PR DESCRIPTION
**Pull request by pokertour**
Hello,

- Add french translation to Cargo, spectator and Zeus
- Cargo : 
-- Add on : <Key ID="STR_ACE_Cargo_paradropTimeCoefficent">
--- <French>Coefficient Temps de larguage de cargaison</French>
-- Add on : <Key ID="STR_ACE_Cargo_paradropTimeCoefficent_description">
-- <French>Modifier le temps qu'il faut pour larguer la cargaison.</French>

- Spectator :
-- Add on : <Key ID="STR_ACE_Spectator_DisplayName">
--- <French>Spectateur ACE</French>

- Zeus : 
-- Add on : <Key ID="STR_ACE_Zeus_ModuleSuppressiveFire_DisplayName">
--- <French>Tir de suppression</French>

### CODE REVIEW

File: addons/cargo/stringtable.xml [hunk 1]
```diff
@@ -259,11 +259,13 @@
             <English>Paradrop Time Coffecient</English>
             <Japanese>空中投下までの時間係数</Japanese>
             <Italian>Coefficente Tempo Lancio Paracadute</Italian>
+            <French>Coefficient Temps de larguage de cargaison</French>
```
Comment (bux) [inline addons/cargo/stringtable.xml]: `largage` ?

**Comment by bux** [comment]
Pinging @alganthe

File: addons/cargo/stringtable.xml (hunk 1 above)
Comment (pokertour) [inline addons/cargo/stringtable.xml]: yes it's "largage" and not "larguage", typping error sry

Comment (bux) [inline addons/cargo/stringtable.xml]: no worries, that's why we have github.

**Comment by alganthe** [comment]
all good,

**Comment by bux** [comment]
@pokertour could you correct the spelling? Then I will merge your PR 👍

**Comment by pokertour** [comment]
@bux done :)

**Pull request merged**